Packages/ClientRuntime/Sources/Networking/Http/SdkHttpClient.swift

```diff
@@ -24,7 +24,9 @@ public class SdkHttpClient {
     }
     
     public func close() {
-        engine.close()
+        Task.detached {
+            await self.engine.close()
+        }
```

Comment:
Wonder why `Task.detached`?

What if you just use `Task`
```
        Task {
            await self.engine.close()
        }
```

Reply:
done